web/src/SyncerEditPage.js

```diff
@@ -47,14 +47,19 @@ class SyncerEditPage extends React.Component {
 
   getSyncer() {
     SyncerBackend.getSyncer("admin", this.state.syncerName)
-      .then((syncer) => {
-        if (syncer === null) {
+      .then((res) => {
+        if (res === null) {
           this.props.history.push("/404");
           return;
         }
 
+        if (res.status === "error") {
```

Comment:
Use `if (res.status === "ok") {` for all places

Reply:
As below comment, this response `error` only, when success it returned entity instead.

Reply:
You mean `if (res.status !== "ok") {` in case error?